src/instructlab/model/download.py

```diff
@@ -341,10 +341,10 @@ def check_skopeo_version():
         if version.parse(installed_version) < version.parse(
             _RECOMMENDED_SCOPEO_VERSION
         ):
-            logger.error(
+            logger.debug(
```

Comment:
This should be a warning
```suggestion
            logger.warning(
```

Reply:
updated, thanks